Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lighting/LightingData.azsli

```diff
@@ -28,6 +28,18 @@ class LightingData
     // Direction light shadow coordinates
     float3 shadowCoords[ViewSrg::MaxCascadeCount];
 
+    // Direction light shadow coordinates for shrinked positions (used in scattering through thin objects)
+    float3 shrinkedShadowCoords[ViewSrg::MaxCascadeCount];
```

Comment:
In the transmission case, are the `shadowCoords` still used or are they replaced by these scaled shadow map queries? Can we alias the memory to keep this structure more compact if possible? (has a direct impact on tiled lighting occupancy)

Reply:
Yes, it can be aliased (or we can also use a local variable, if needed in the future, see https://github.com/o3de/o3de/pull/6428/commits/6dbb8e1794f763302226dce41ff91e99b3b7e875). Aliasing done in https://github.com/o3de/o3de/pull/6428/commits/60ec8c3cfb5d9a1110a96977bdc3318739b3f1e0.